routes/tasks.go

```diff
@@ -0,0 +1,123 @@
+package routes
+
+import (
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/labstack/echo"
+	"net/http"
+
+	"github.com/kirikami/go_exercise_api/database"
+	u "github.com/kirikami/go_exercise_api/utils"
+)
+
+func (h ApiV1Handler) SaveTaskHandler(c echo.Context) error {
+	task := database.Task{}
+
+	err := c.Bind(&task)
+
+	if err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest)
+	}
+
+	err = h.DB.Save(&task).Error
+	if err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest)
+	}
+
+	return c.JSON(http.StatusCreated, task)
+}
+
+func (h ApiV1Handler) UpdateTaskHandler(c echo.Context) error {
+	idParam := c.P(0)
```

Comment:
then problem with goth library mb, when I set address like /callback/:provider it couldn't read parameter only as /calback?provider=facebook